pubspec.yaml

```
@@ -57,13 +57,21 @@ dependencies:
   firebase_storage: ^12.4.5
   cached_network_image: ^3.4.1
   path: ^1.9.0
+  fake_cloud_firestore: ^3.0.2
 
 
   #------------/. PACKAGES END ---------------#
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)